test-suite/shared/src/test/scala/org/scalajs/testsuite/compiler/FloatTest.scala

```diff
@@ -0,0 +1,45 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js Test Suite        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013, LAMP/EPFL        **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+package org.scalajs.testsuite.compiler
+
+import org.junit.Test
+import org.junit.Assert._
+
+class FloatTest {
+  @Test
+  def `Float_toInt`(): Unit = {
```

Comment:
`Float_toInt` looks redundant to me, since we're in `FloatTest`. I suggest just `toInt`. If you agree, we should follow the same convention everywhere.